docs/designs/data_transform.md

```diff
@@ -210,64 +224,101 @@ COLUMN
 LABEL label
 ```
 
-SQLFlow will convert the `COLUMN` expression to Python code of data transformation. But it requires some parameters which are derived from the data. So next we will do the analysis work.  
+SQLFlow will compile the `COLUMN` expression to Python code of data transformation. Let's take the SQLFlow statement above for example to describe the compilation workflow step by step in the next section.  
 
-### Generate Analysis SQL From SQLFlow Statement
+### COLUMN Expression Compilation Workflow
 
-SQLFlow will generate the analysis SQL to calculate the statistical value. For this clause `COLUMN NORMALIZE(capital_gain), STANDARDIZE(age)`, the corresponding analysis SQL is as follows:
+#### Parse COLUMN Expression into AST
+
+Given a COLUMN expression, syntax parsing is naturally the first step. We will parse one COLUMN expression to one AST. For the SQL statement containing two column expressions above, we will parse them into two ASTs. Please check the following figure of the AST for the example SQL.  
+![column_clause_ast](../images/column_clause_ast.png)
+
+#### Convert the AST into a DAG of Transform Flow
```

Comment:
I am afraid that SQLFlow extended syntax parser doesn't generate AST. 
Would this fact affect the design here?

Reply:
I think it won't affect the design.
What we want is that we can convert the `COLUMN` clauses into the transform flow DAG here. Maybe we can try convert the IR into the DAG.